### PR TITLE
Jared/user detail styles

### PR DESCRIPTION
**Pull request by jaredcunha-usds**
**Why**

This ended up being a larger PR to style more parts of the internal user experience since a lot of the pages are closely related. 

**What Changed**

**users#show**
Screen Shot 2019-10-03 at 1 58 26 PM

**users#edit**
Screen Shot 2019-10-03 at 1 58 34 PM

**organizations#new**
Screen Shot 2019-10-03 at 1 58 55 PM

**organizations#show**
Screen Shot 2019-10-03 at 1 59 05 PM

**organizations#edit**
Screen Shot 2019-10-03 at 2 04 24 PM

**Choices Made**

Aside from the new pages, the `filter-group` was changed to a `field-group` because it is more generic and thus reusable. Semantic tags were also put into place.

### CODE REVIEW

File: dpc-web/app/views/internal/organizations/edit.html.erb [hunk 1]
```diff
@@ -1,15 +1,18 @@
...
-    <div class="ds-u-border--1 ds-u-padding--2 ds-u-radius">
+    <h1 class="ds-u-font-size--title ds-u-font-weight--normal ds-u-margin--0">Edit: <%= @organization.name %></h1>
+
+  </section>
+
+  <section class="box">
+    <div class="box__content">
       <%= form_for @organization, url: internal_organization_path(@organization), html: { method: :put } do |f| %>
-        <%= render 'internal/organizations/fields', f: f %>
+        <%= render 'internal/organizations/fields', f: f, locals: {:button_label => "Save"} %>
```
Comment (switzersc-usds) [inline dpc-web/app/views/internal/organizations/edit.html.erb]: hash rocket! vote to change `:button_label => ` to `button_label:`

File: dpc-web/app/views/internal/organizations/new.html.erb [hunk 2]
```diff
@@ -1,15 +1,19 @@
...
-    <div class="ds-u-border--1 ds-u-padding--2 ds-u-radius">
+    <h1 class="ds-u-font-size--title ds-u-font-weight--normal ds-u-margin--0">Create a new provider organization</h1>
+
+  </section>
+
+  <section class="box">
+    <div class="box__content">
       <%= form_for @organization, url: internal_organizations_path do |f| %>
-        <%= render 'internal/organizations/fields', f: f %>
+        <%= render 'internal/organizations/fields', f: f, locals: {:button_label => "Create organization"}%>
```
Comment (switzersc-usds) [inline dpc-web/app/views/internal/organizations/new.html.erb]: ditto

File: dpc-web/app/views/internal/organizations/edit.html.erb (hunk 1 above)
Comment (jaredcunha-usds) [reply dpc-web/app/views/internal/organizations/edit.html.erb]: Done

File: dpc-web/app/views/internal/organizations/new.html.erb (hunk 2 above)
Comment (jaredcunha-usds) [reply dpc-web/app/views/internal/organizations/new.html.erb]: Also done

**Pull request merged**